src/main/java/seedu/lifeasier/commands/EditNotesCommand.java

```diff
@@ -48,16 +49,16 @@ private void findTitle(Ui ui, NoteList notes, Parser parser,
             ui.showMultipleMatchesFoundMessage();
 
             logger.log(Level.INFO, "Start of printing all matching notes");
-            ui.printMultipleNoteMatches(notes, title);
+            matchArr = ui.printMultipleNoteMatches(notes, title, matchArr);
             logger.log(Level.INFO, "End of printing all matching notes");
 
-            noteNumber = Integer.parseInt(ui.readCommand()) - 1;
-            NoteCommandFunctions.checkForIndexOutOfBounds(notes, noteNumber);
+            noteNumber = Integer.parseInt(ui.readCommand());
```

Comment:
Same comment about the "-1" logic. I'm not sure if that'll help to make it clearer!